tests/installation/livecdreboot.pm

```diff
@@ -79,9 +79,12 @@ sub run() {
     # 2nd stage automatic configuration. And then ere is also
     # reboot_after_install from 800_reboot_after_install.pm
     # should assert_screen wait for all three at the same time and then have only check_screen afterwards?
-    my $ret = assert_screen [qw(grub2 second-stage)], 120;
-    if ($ret->{needle}->has_tag('grub2')) {
-        send_key "ret";    # avoid timeout for booting to HDD
+    if (!get_var('distri') == 'opensuse') {
```

Comment:
well, either use 'eq' instead of '==' or go with check_var() :)
hmm...maybe consider to use !get_var("UPGRADE") or even and !get_var("ZDUP") in case? I can't see any reason to check distri here, SLE and openSUSE should be same behavior, and the others distribution should not use this test case.